interface/main/calendar/modules/PostCalendar/pntemplates/default/views/day/ajax_template.html

```diff
@@ -151,50 +151,65 @@
 //==================================
 
 [-/php-]
-<div id="topToolbarRight" class="bgcolor2">  <!-- this wraps some of the top toolbar items -->
-<div id="functions">
-<!-- stuff form element here to avoid the margin padding it introduces into the page in some browsers -->
-<form name='theform' id='theform' action='index.php?module=PostCalendar&func=view&tplview=default&pc_category=&pc_topic=' method='post' onsubmit='return top.restoreSession()'>
-<input type="hidden" name="jumpdate" id="jumpdate" value="">
-<input type="hidden" name="viewtype" id="viewtype" value="[-php-]echo $viewtype;[-/php-]">
-[-php-]
-echo "   <a href='#' title='" . xl("New Appointment") . "' onclick='newEvt(1, 9, 00, $Date, 0, 0)' class='css_button'>
-            <i class='fa fa-plus' aria-hidden='true'></i></a>\n";
-echo "   <a href='#' title='" . xl("Search Appointment") . "'
-            onclick='top.restoreSession();location=\"index.php?module=PostCalendar&func=search\"' class='css_button'>
-            <i class='fa fa-search' aria-hidden='true'></i></a>\n";
-[-/php-]
-</div>
-
-
-<div id="dateNAV"">
-[-php-]
-if ($Date <> date('Ymd')) {
-[-/php-]
-<a href='#' name='bnsubmit' value='[-php-]echo xl ("Today")[-/php-]' onClick='GoToToday(theform);'  class='css_button'/><span>[-php-] echo xl("Today")[-/php-]</span></a>
-[-php-]
-}
-echo "   <a id='prevday' href='$PREV_DAY_URL' onclick='top.restoreSession()' title='" . xl("Previous Day") . "'>
-            <i class='fa fa-arrow-left' aria-hidden='true'></i></a>\n";
-$atmp = array_keys($A_EVENTS);
-echo dateformat (strtotime($atmp[0]),true);
-echo "   <a id='nextday' href='$NEXT_DAY_URL' onclick='top.restoreSession()' title='" . xl("Next Day") . "'>
-            <i class='fa fa-arrow-right' aria-hidden='true'></i></a>\n";
-[-/php-]
-</div>
+[-php-]require_once "{$GLOBALS['srcdir']}/templates/standard_header_template.php";[-/php-]
```

Comment:
This does not seem to be the right space for this. Recommend checking out the source of the calendar output to track down right upstream place for this(as I recall, there are nice comments that can be grepped to find the upstream script). Note the variables in standard_header_template.php needed to be changed to globals or else they don't show up in smarty code (note I just fixed this in the commit I brought in for the SOAP form :) ).